UnitsNet/UnitDefinitions/MassFlow.json

```diff
@@ -32,6 +32,18 @@
         }
       ]
     },
+    {
+      "SingularName": "TonPerHour",
```

Comment:
Rename this to `TonnePerHour` to match the same unit in `Mass`. I believe `ton` is the American naming.

Reply:
Fixed, thank you.